axelrod/player.py

```diff
@@ -98,6 +98,15 @@ def play(self, opponent, noise=0):
             s1, s2 = self._add_noise(noise, s1, s2)
         update_histories(self, opponent, s1, s2)
 
+    def reproduce(self):
```

Comment:
Oh yes, now I remember. I would say generally that is MindReader's problem and he should have a custom __copy__ method to prevent that. But I do soo the point of this... maybe rename to `clone` though?